lib/api/src/externals/function.rs

```diff
@@ -478,16 +480,23 @@ impl Function {
         }
 
         // Call the trampoline.
-        if let Err(error) = unsafe {
-            wasmer_call_trampoline(
-                &self.store,
-                self.exported.vm_function.vmctx,
-                trampoline,
-                self.exported.vm_function.address,
-                values_vec.as_mut_ptr() as *mut u8,
-            )
-        } {
-            return Err(RuntimeError::from_trap(error));
+        unsafe {
```

Comment:
I think the unsafe block can be moved into just one of the branches here, that makes it more obvious what could be going wrong here